diagnostic.md

```diff
@@ -93,10 +110,15 @@ Write the `curl` request you'd use to change the donut. We'll send the following
 JSON. Please do not use `data = getFormFields(form)` instead write out the data
 object.
 
-```json
+```curl --include --request PATCH \
+"http://www.example.com" \
+--header "Content-Type: application/json" \
+--data
 {
   "name": "Krüller"
-}
+}''
+
+
 ```
```

Comment:
your curl request is right, but just so you know the way to use the snippet in markdown would be:

```md
```json
curl --include --request PATCH \
 "http://www.example.com" \
 --header "Content-Type: application/json" \
 --data
  {
    "name": "Krüller"
 }''
<```>Just the three ``` to close the ```json above
```